apps/web/src/config/index.ts

```diff
@@ -68,3 +68,7 @@ export const IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = isCypress
   : window._env_.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED ||
     process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED ||
     'false';
+
+export const IS_MULTI_TENANCY_ENABLED = isCypress
+  ? window._env_.IS_MULTI_TENANCY_ENABLED || process.env.IS_MULTI_TENANCY_ENABLED || 'true'
```

Comment:
Was not sure if to call it multi-tenancy or just tenant/tenants, naming 🤦‍♂️

Reply:
I think it sounds good